File: beer-store-service-java-spring-boot/src/main/java/beer/store/model/Beer.java
package beer.store.model;

import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import lombok.Data;

@Data
@Entity
@Table(name = "beer")
public class Beer {

  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  @Column(name = "idbeer")
  private Long idBeer;

  @Temporal(TemporalType.TIMESTAMP)
  @Column(name = "creationdatebeer")
  private Date creationDateBeer;

  @Column(name = "titlebeer")
  private String titleBeer;

  @Column(name = "descriptionbeer")
  private String descriptionBeer;

  @Column(name="idmedia")
  private Long idMedia;

}

File: beer-store-client-kotlin-android/app/src/main/java/beer/store/Beer.kt
package beer.store

import com.google.gson.annotations.SerializedName
import java.util.*

data class Beer(
    @SerializedName("idbeer")
    val idBeer: Int = 0,
    @SerializedName("titlebeer")
    val titleBeer: String = "Sample beer",
    @SerializedName("creationdatebeer")
    val creationDateBeer: Date = Date(),
    @SerializedName("descriptionbeer")
    val descriptionBeer: String = "Sample empty bottle beer",
    @SerializedName("idmedia")
    val idMedia: Int = 0
)